footer.php

```diff
@@ -18,12 +18,15 @@
 
 						<ul class="footer-menu reset-list-style">
 							<?php
-							wp_nav_menu( array(
-								'container'      => '',
-								'depth'          => 1,
-								'items_wrap'     => '%3$s',
-								'theme_location' => 'footer-menu',
-							) );
+
+							wp_nav_menu(
+								array(
```

Comment:
You should be able to `wp_nav_menu( array(`

See https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#indentation

I think the `array(` is part of PHPCBF automatically adding it to its own line which is not required, having `array(` on its own line ends up a little too much whitespace.

Denis can you update this instance, and if it passes the PHPCS check update all other instances of this too please

Reply:
Putting the array after the function call triggers the `PEAR.Functions.FunctionCallSignature.ContentAfterOpenBracket` sniff, so this and `PEAR.Functions.FunctionCallSignature.CloseBracketLine` are good candidates for exclusions in the `phpcs.xml.dist`

Reply:
Agreed that this could result in a lot of whitespace that does not lend itself any better to readability than having the array on the function line.

Reply:
We could just allow this: https://github.com/WordPress/WordPress-Coding-Standards/blob/ecb633d8cddb5fa38dc2de5d72b220810dde549c/WordPress-Core/ruleset.xml#L246